common/src/test/java/com/revenuecat/purchases/common/BackendTest.kt

```diff
@@ -1925,7 +1925,7 @@ class BackendTest {
                 )
                 assertThat(it).isEqualTo(expectedMapping)
             },
-            { fail("expected success") }
+            { error -> fail("expected success $error", error) }
```

Comment:
Was this needed for debugging?

Reply:
Yeah this was failing so I had to add it to see the actual error. I figured it wouldn't hurt to leave it